app/star-rating.jsx

```diff
@@ -31,6 +31,7 @@ class Star extends React.Component {
   onChange(e) {
      const rating = parseInt(e.target.value);
      this.props.onChange(rating);
+     console.log('shows checked is true on click', this);
```

Comment:
Remove this.

Reply:
see commit "removes console log statement"